tests/components/CreateWallet.test.js

```diff
@@ -1,18 +1,16 @@
 import { mount } from '@vue/test-utils';
-import { nextTick, ref } from 'vue';
 import { expect } from 'vitest';
 import CreateWallet from '../../scripts/dashboard/CreateWallet.vue';
 import Modal from '../../scripts/Modal.vue';
 import { vi, it, describe } from 'vitest';
-import * as settings from '../../scripts/settings';
 
 describe('create wallet tests', () => {
-    afterEach(() => vi.clearAllMocks());
     it('Generates wallet', async () => {
-        // mock settings to disable advancedmode
-        vi.spyOn(settings, 'fAdvancedMode', 'get').mockReturnValue(false);
-
-        const wrapper = mount(CreateWallet);
+        const wrapper = mount(CreateWallet, {
```

Comment:
```suggestion
        const wrapper = shallowMount(CreateWallet, {
```
Since we're not testing the child components directly (which is a good thing, we don't want to test too many things at once) you can use `shallowMount` to stub all the children automatically

Reply:
Sorry wrong test, this comment should be under `LoginTest.test.js`